alacritty/src/config/font.rs

```diff
@@ -22,6 +22,9 @@ pub struct Font {
     /// Glyph offset within character cell.
     pub glyph_offset: Delta<i8>,
 
+    #[config(removed = "To enable thin strokes on macOS, run:\n$ defaults write -g \
+                        AppleFontSmoothing -int 0\nand relaunch Alacritty. Use io.alacritty \
+                        instead of -g to enable thin strokes in Alacritty only.")]
```

Comment:
I don't think you'd need new lines for those actually. I'm pretty sure that our message bar will format everything properly in that case. Could you try it and see how it looks? Also, I don't think we should document how to use it right away. I'd suggest to mention the option that should be used, not how it should be used.

Reply:
I tried doing it a few different ways, but I also have to get it to pass `rustfmt`, so I need to use the `\` line continuations in order to align the start of each line of the string literal, which forces the use of `\n` for actual newlines in the message.

Reply:
My concern is that people will not know where to look, and at least some of them may continue to file duplicate issues or pile on to existing ones if they (reasonably) try to set it to `false`/`NO`, or can't figure out how the `defaults` command works.

Reply:
If I remove the bit about `io.alacritty` vs. `-g` do you think it would be sufficiently short?

Reply:
if you run `cargo +nightly fmt` won't it format it?

Reply:
>I tried doing it a few different ways, but I also have to get it to pass rustfmt, so I need to use the \ line continuations in order to align the start of each line of the string literal, which forces the use of \n for actual newlines in the message.

I thought that our message bar will add those automatically or expand across the strings? I'm not sure you'd need an explicit `\n` for those.

Reply:
Yes, my changes now are `fmt`'ed. My previous attempts failed the build because they did not pass `rustfmt --check`. I'm not quite sure what you'd like me to test regarding newlines. The message now renders as I expect it to, but if I remove the newlines, it'll all be on one line and will wrap differently depending on the window width.

Reply:
How about this?
```rs
    #[config(removed = "To enable thin strokes on macOS, run:\n\
                        $ defaults write -g AppleFontSmoothing -int 0\n\
                        and relaunch Alacritty.")]
```

Reply:
>The message now renders as I expect it to, but if I remove the newlines, it'll all be on one line and will wrap differently depending on the window width, and likely splits the command across lines.

Exactly, your case could generate arbitrary new lines, I'd suggest to do that without `\n` and suggest to use `org.alacritty` for settings instead of making it global. After that it should be good to go.

Reply:
> I thought that our message bar will add those automatically or expand across the strings? I'm not sure you'd need an explicit \n for those.

How do I tell it where the line breaks should go? The warning log is just one string, and it will be one line if there are no newlines. Here is what that looks like for me, with the following config attribute:
```rs
    #[config(removed = "To enable thin strokes on macOS, run: `$ defaults write -g \
                        AppleFontSmoothing -int 0` and relaunch Alacritty.")]
```
<img width="801" alt="removal_warning_oneline" src="https://user-images.githubusercontent.com/2592855/178161922-78ccf18f-07e0-4454-a950-c6220316404e.png">

Reply:
> Exactly, your case could generate arbitrary new lines, I'd suggest to do that without \n and suggest to use org.alacritty for settings instead of making it global. After that it should be good to go.

Oh, I thought you were saying there was some special thing the message bar would do here with actual newlines vs. newline escape sequences. If you just don't want a multi-line log message, that's fine with me, but I thought it would be more readable and easier to select if the command is guaranteed to be on its own line, which is achieved by doing explicit newlines.

Reply:
Take 3. What do you think? This is with 100 columns.
<img width="801" alt="removal_warning_take_3" src="https://user-images.githubusercontent.com/2592855/178162349-d6d8ef61-03ec-4609-8fbc-ecb90dddf776.png">

Reply:
And with the newline (and using `org.alacritty`, per your recent update).
<img width="801" alt="removal_warning_take_4" src="https://user-images.githubusercontent.com/2592855/178162671-4726ee47-0b76-4c17-9d8f-176d77b1b000.png">

Reply:
Rebased and pushed the latter just now. If you really don't like the explicit newline I can remove it... This whole thing will be removed eventually anyway, though.

Reply:
>to select if the command is guaranteed to be on its own line, which is achieved by doing explicit newlines.

You can't select on message bar, so it doesn't matter.

Reply:
> You can't select on message bar, so it doesn't matter.

ooof... whoops. Let me rethink this then. Sorry for the churn here.